lib/src/test/java/com/auth0/jwt/JWTCreatorTest.java

```diff
@@ -507,7 +508,7 @@ public void shouldAcceptCustomMapClaimOfBasicObjectTypes() throws Exception {
         data.put("integer", 1);
         data.put("long", Long.MAX_VALUE);
         data.put("double", 123.456d);
-        data.put("date", new Date(123L));
+        data.put("date", new Date(123000));
```

Comment:
`java.util.Date` claims should be serialized to the seconds since the epoch. `Date` constructor takes a `long` representing the milliseconds since the epoch.